### DISCUSSION TITLE
When will nelm be available as a standalone CLI?

### DISCUSSION THREAD

**Discussion by waddles** [werf/nelm#120] [Q&A] [answered]
We are interested in using nelm as a deployment tool to replace helm. The README.md indicates it will be available as a standalone tool but gives no indication of when or what needs to be done to achieve that.

**Comment by shurup** [comment]
I do appreciate your interest!

There is a small team behind this project, and we're currently busy implementing Nelm Actions. They will open the path for making Nelm CLI and custom integrations that need an API (e.g., creating a Flux controller using Nelm, which is also in our plans).

We wanted to release Nelm CLI earlier, but realistically (according to our current resources), we can expect it at best in Q4 of this year.

**Comment by ilya-lesikov** [comment, marked as answer]
Nelm 1.0 is released, along with a standalone CLI. Follow instructions in the README: https://github.com/werf/nelm
